frontend/packages/dev-console/src/components/import/devfile/DevfileImportForm.tsx

```diff
@@ -45,26 +45,27 @@ const DevfileImportForm: React.FC<FormikProps<FormikValues> & DevfileImportFormP
             {devfileParseError}
           </Alert>
         )}
-        <FormSection>
-          {formType === 'sample' ? (
+        {formType === 'sample' ? (
+          <FormSection>
```

Comment:
as there is conditional rendering based on formType, I think it'll make sense to add spec for it. WDYT?

Reply:
Added tests for this component